assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.js

```diff
@@ -236,9 +239,10 @@ function DashboardIdeasWidget( props ) {
 				DashboardIdeasWidget.tabIDsByIndex[ tabIndex ]
 			);
 
+			clearErrors( 'updateIdeaState' );
```

Comment:
@kuasha420 just call the `clearErrors()` function without parameters:
```suggestion
			clearErrors();
```